layersvt/device_simulation.md

```diff
@@ -135,19 +135,28 @@ If you wish DevSim to terminate on errors, set the `VK_DEVSIM_EXIT_ON_ERROR` env
 }
 ```
 
+### `VK_KHR_portability_subset` Emulation
+
+The DevSim layer provides the ability to emulate the `VK_KHR_portability_subset` extension on devices that do not implement this extension.
+This feature allows users to test their application with limitations found on non-conformant Vulkan implementations.
+To turn on this feature, set the `VK_DEVSIM_EMULATE_PORTABILITY_SUBSET_EXTENSION` environment variable or `lunarg_device_simulation.emulate_portability`, the corresponding vk_layer_settings.txt option to a positive integer.
+DevSim config files that use this feature should validate to the following schema (link).
+
 ### Environment variables used by DevSim layer.
```

Comment:
To emphasize the availability and ease of use of the Vulkan Configurator, I believe a section needs to be added to this document referencing use of the Vulkan Configurator to enable the devsim layer and configure it.

Reply:
I am wondering if it is a better organization to talk about the environment variables and vk_layer_settings.txt options in the same section in the document, vs. pulling this into a separate section for the portability extension.

Also, shouldn't you update the DevSim Layer operation and configuration section with information on the new schema for the portability extension (indicate the schema presence and location)?